asr-improve-recognition-for-specific-words.md

```diff
@@ -22,11 +22,11 @@ To improve the recognition of specific words, use the following customizations.
 
 |              Techniques              |    Difficulty   |                                                  What it does                                                  |                                         When to use                                        |       How to use       |
 |:------------------------------------:|:---------------:|:--------------------------------------------------------------------------------------------------------------:|:------------------------------------------------------------------------------------------:|:----------------------:|
-| 1. Word boosting                        | Quick and easy  | Temporarily extend the vocabulary while increasing the chance of recognition for a provided list of words.              | When you know that certain words or phrases are important.                                 | [Tutorial](https://github.com/nvidia-riva/tutorials/blob/main/asr-wordboosting.ipynb) |
+| 1. Word boosting                        | Quick and easy  | Temporarily extend the vocabulary while increasing the chance of recognition for a provided list of words.              | When you know that certain words or phrases are important.                                 | [Tutorial](https://github.com/nvidia-riva/tutorials/blob/main/asr-wordboosting.ipynb), [ZH_Tutorial](https://github.com/nvidia-riva/tutorials/blob/main/Language-Scaling/Mandarin/asr-wordboosting-zh.ipynb) |
 | 2. Custom vocabulary                    | Easy            | Permanently extend the default vocabulary to cover novel words of interest.                                             | When the default model vocabulary does not sufficiently cover the domain of interest.      | [Tutorial](https://github.com/nvidia-riva/tutorials/blob/main/asr-customize-vocabulary-and-lexicon.ipynb) |
 | 3. Custom pronunciation (Lexicon mapping)                      | Easy            | Explicitly guide the decoder to map one or more pronunciations (sequences of tokens) to a specific word.                                          | When you know a word can have multiple pronunciations.                            | [Tutorial](https://github.com/nvidia-riva/tutorials/blob/main/asr-customize-vocabulary-and-lexicon.ipynb) |
-| 4. Retrain language model               | Moderate        | Train a new language model for the application domain to improve the recognition of domain specific terms.     | When domain text data is available.                        | Training Tutorial Coming Soon |
```

Comment:
"Training Tutorial Coming Soon" - should this text be removed?

others two links are fine.

Reply:
This is the original text in the tutorial, and I'm working on ZH retraining LM. Actually, it will be available soon ^^

Reply:
Right, so we should keep it as "Training Tutorial Coming Soon". In your latest changes, this text is removed.

Reply:
Got it. Let me check

Reply:
Done